content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```diff
@@ -0,0 +1,558 @@
+---
+date: '2023-06-20'
+version: 'Corda 5.0 Beta 4'
+title: "Vault Queries"
+menu:
+  corda5:
+    identifier: corda5-api-vault-queries
+    parent: corda5-api-ledger-utxo
+    weight: 3000
+section_menu: corda5
+---
+
+# Vault Queries
+
+## Vault Named Query
+
+A vault named query is a database query that can be defined by Corda users. The user can define the following:
+
+* The name of the query
+* The query functionality (state type the query will work on and the `WHERE` clause)
+* Optional filtering logic of the result set
+* Optional transformation logic of the result set
+* Optional collection logic of the result set
+* The query creator needs to follow a few pre-defined steps in order for the query to be registered and to be usable.
```

Comment:
Doesn't seem like it should be part of the list? Or maybe it just needs to be rewritten?